### DISCUSSION TITLE
Dexcome One+ Companion App

### DISCUSSION THREAD

**Discussion by Steppasaurus** [NightscoutFoundation/xDrip#3425] [Q&A]
Apologies if this has been asked and or answered, I did search but couldn't find much related to the Dexcom One+ and wanting to know if it's possible to use in companion app mode? I have just moved from the G7 to the One+ which in theory is the same hardware, but companion app mode doesn't work (it was working fine with the G7). Wondering if I've missed something or it's just not supported yet.

At the moment I've reconfigured it to pull data from Dexcom Share instead, which works fine for now but if I'm in an area with low coverage or no data, then having xDrip pull directly from the Dexcom app works perfectly so hoping I can get that working again.

Any info much appreciated!

**Comment by Navid200** [comment]
Please post a screenshot of a Dexcom One+ app notification.

**Reply by Steppasaurus** [reply]
Screenshot attached. 

![Screenshot_20240409_080311_One UI Home](https://github.com/NightscoutFoundation/xDrip/assets/112546110/449b4ea2-2ac6-4e23-b82f-4252b5aa972a)

**Reply by Navid200** [reply]
Are you able to test this?
https://github.com/NightscoutFoundation/xDrip/pull/3426

**Comment by Steppasaurus** [comment]
Hi Navid,

Thanks so much for your reply but I am very novice at GitHub, how do I
actually download your custom APK to test?

**Reply by Navid200** [reply]
Never mind what I asked.
Please update to the latest Nightly, which is this:
https://github.com/NightscoutFoundation/xDrip/releases/tag/2024.04.09

I have explained how update works here: https://navid200.github.io/xDrip/docs/Updates.html

Please let me know if you need help with updating.

After you update, please tell me if it works or not.

**Comment by Steppasaurus** [comment]
Hi Navid,

Ok perfect thanks for that. I have updated to nightly and confirmed it is
now working fine, companion app is now reading from Dexcom One+ absolutely
fine.

Thanks for this!

**Reply by Navid200** [reply]
Great,
Thanks

And you're welcome
